Solutions/PaloAltoCDL/Hunting Queries/PaloAltoCDLRarePortsbyUser.yaml

```diff
@@ -16,7 +16,7 @@ query: |
   PaloAltoCDLEvent
   | where TimeGenerated > ago(24h)
   | where isnotempty(DstPortNumber) 
-  | summarize RarePorts = count() by DstPortNumber, DstUsername
+  | summarize RarePorts = count() by DstPortNumber, DstIpAddr, DstUsername

```

Comment:
Considering that we are also adding an IP address in the output , might be a good idea to consider adding the IP entity to the entity mapping of this query.

Reply:
added ip entity